.circleci/config.yml

```diff
@@ -612,6 +612,11 @@ jobs:
           command: |
             just snapshots-check || echo "export SNAPSHOTS_STATUS=1" >> "$BASH_ENV"
           working_directory: packages/contracts-bedrock
+      - run:
```

Comment:
This is getting to be huge, I'd like to refactor this in the future so its just a single script in CI and then we call all of these things inside of the script

Reply:
I had a different suggestion in https://github.com/ethereum-optimism/optimism/issues/11722 if you want to move the convo there, I'm good with that approach too as long as we can still run/log all checks at once

Reply:
Ok let's address this as part of a different PR